Ghidra/Features/Decompiler/src/decompile/cpp/varmap.hh

```diff
@@ -140,6 +141,7 @@ private:
   AddrSpace *space;		///< AddressSpace in which to search
   mutable vector<AddBase> addBase; ///< Collection of pointers into the AddressSpace
   mutable vector<uintb> alias;	///< List of aliased addresses (as offsets)
+  mutable unordered_map<uint4, uintb> calculatedOffsets; ///< Map of offsets of already visited Varnodes so the recursive algorithm doesn't blow the stack
```

Comment:
It is probably better to use `std::map`  here (red black tree based) instead of `std::unordered_map` since the key is a 32 bit unsigned integer.